Code/MinimalLib/README.md

```diff
@@ -1,11 +1,139 @@
+
+The idea of this package is to allow the [RDKit](https://github.com/rdkit/rdkit) to be used from JavaScript so that we can add chemical capabilities to web applications.  
+
+Rather than attempting a comprehensive wrapper (like the old [RDKitJS](https://github.com/rdkit/RDKitjs)), this exposes a small set of key functionality. I think the general approach, including this actual library, can be useful for other wrapper projects in the future.
 
 This initial set of functionality is not complete, but it is intended to already be directly useful.
 
-The `Dockerfile` in the `docker/` shows how to setup an appropriate environment and build the wrappers.
+## Install
+
+```bash
+npm i @rdkit/rdkit
+```  
+
+## Using the RDKit package assets
+
+### Option 1: Use the npm package distribution files
+
+Once you have the RDKit package installed in your node modules, copy the following distribution files anywhere in your deployed assets.
+
+- `node_modules/@rdkit/rdkit/CodeMinimalLib/dist/RDKit_minimal.js`
+- `node_modules/@rdkit/rdkit/CodeMinimalLib/dist/RDKit_minimal.wasm`
+
+**NOTE: Both files must be copied at the same location in your deployed assets for the library to work properly.**
+
+### Option 2: Use the remote distribution files from [unpkg.com](https://unpkg.com/)
+
+- `https://unpkg.com/@rdkit/rdkit@2021.3.1-beta.13/Code/MinimalLib/dist/RDKit_minimal.js`
+- `https://unpkg.com/@rdkit/rdkit@2021.3.1-beta.13/Code/MinimalLib/dist/RDKit_minimal.wasm`
+
+## Running RDKit in your JavaScript code
+
+To use RDKit, load the javascript file and instantiate the wasm module inside the `head` tag of your `index.html`, before you run your application code:
+
+```html
+<head>
+    <!-- ...other files and HTML tags... -->
+    <!-- Load the RDKit JS file -->
+    <script src="https://unpkg.com/@rdkit/rdkit@2021.3.1-beta.13/Code/MinimalLib/dist/RDKit_minimal.js"></script>
+    <!-- Instantiate the WASM module. The inline script below could live elsewhere inside your application code. -->
+    <script>
+        window.initRDKitModule()
+            .then(function(RDKit) {
+                window.RDKit = RDKit;
+                /**
+                 * The RDKit module is now loaded.
+                 * You can use it anywhere.
+                 */
+            })
+            .catch(() => {
+                // handle loading errors here...
+            });
+    </script>
+    <!-- ...your application code goes here... -->
+</head>
+
+```
+
+## Usage
+
+The best examples on how you can use RDKit are currently at https://rdkit.org/rdkitjs/beta/GettingStartedInJS.html .
+
+Inspect the source code of this page to see the various ways to use the JavaScript release of RDKit.
+
+## Live demos
+
+- Getting Started: https://rdkit.org/rdkitjs/beta/GettingStartedInJS.html
+- From this npm package: https://unpkg.com/@rdkit/rdkit@2021.3.1-beta.13/Code/MinimalLib/dist/demo.html
```

Comment:
I guess this will need to change for the production version.

Reply:
Thanks Dave, yes it will :+1: - I already switched it to the last beta release